### ISSUE TITLE
Series selection in previously opened mpReview processed directory

### ISSUE THREAD

**Issue by che85** [fedorov/PCampReview#160]
mpReview should select the exact same series as selected by the user previously when reopening/reloading a mpReview processed directory instead of selecting the default over and over again. 

TODO: need to think of a way to save it. Maybe we could have something like a directory specific json file which also could keep track of applied W/L values for latter loading

@fedorov Thoughts on this?

**Comment by fedorov** [comment]
Yes, I think it makes sense to include a json file with this kind of data. I already had a JSON with W/L information some time ago. We should ask David if there is a mechanism in DICOM to store this kind of information.

I was also thinking it would make sense to have the functionality to assign "series type" in mpReview implemented similarly to how we tag series in SliceTracker. Perhaps the same set of components can be reused (automatic assignment of the series type, with the user having the capacity to re-assign via a drop-down), and the rules could be defined similarly. What do you think?

**Comment by che85** [comment]
Sounds good. What "series types" were you thinking of?

**Comment by fedorov** [comment]
https://github.com/fedorov/mpReviewUtil/blob/master/SeriesNormalizer.py#L58

Turns out, I never cleaned up and integrated this ... We should discuss how much makes sense to do in mpReview, and what should be done for the PI-RADS workflow implementation. Need to think about this!

**Comment by fedorov** [comment]
@che85 just realized this - can you please test if this also works while creating pmap? It should, since the changes applied to the component shared between seg and pmap, but it's good to confirm this.
